kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupSqlDefault.java

```diff
@@ -1,511 +1,516 @@
-/**********************************************************************************

```

Comment:
So for the life of me, I can't seem to modify this file "properly". If I edit the file in my IDE with the proper whitespace setting for the file (tabs), as soon as I add a line anywhere in the file, it reports the whole file as being changed.

If I edit the file on the command line with emacs, the diff produced will be only the lines I added, but the whitespace is wrong (uses spaces instead of tabs).

Any suggestions?